### PR TITLE
Spring and MySQL local configuration tutorial and implementation (Windows Portion)

### PR DESCRIPTION
**Pull request by TonyLiu836**
Add in Windows download & installation configuration portion of tutorial. #13

### CODE REVIEW

**Review by Junzhou-712 — APPROVED** [review]
LGTM! Approved.

File: Docs/Tutorials&Mannuals/Spring and MySQL local configuration tutorial.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        ![MySQL workbench](../Resources/MySQL.png)
+      - Click the button in the picture and create the databse with the name you want. 
+      - Remember, you do not need to create any table or insert any data into it, we are using JPA, JPA will create tables according to our code implementation.
+
+### 2.2 Windows 10/11
+Download and run MySQL Installer from here: https://dev.mysql.com/downloads/installer/
+- Choose "Full installation." Should install the following itms
+    ![Win11 MySQL Installation Page](../Resources\MySQL_install_screenshot.png)
+- Choose default for everything until the following screen is reached. Make sure all fields match with the image below.
+    ![MySQL Type & Networking](../Resources/MySQL_post_installation.png)
```
Comment (RunpengNie) [inline Docs/Tutorials&Mannuals/Spring and MySQL local configuration tutorial.md]: The picture links are not working.

**Review by RunpengNie — CHANGES_REQUESTED** [review]

**Comment by RunpengNie** [comment]
The PR's `Assignees` should be empty or only you, `Reviewers` should be the other developers. 
Also, I have updated my PR, please remember to do a git pull to avoid future merge conflicts.

**Review by RunpengNie — APPROVED** [review]
LGTM!

**Pull request merged**